pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdNamespaces.java

```diff
@@ -325,6 +325,18 @@ void run() throws PulsarAdminException {
         }
     }
 
+    @Parameters(commandDescription = "Remove replication clusters for a namespace")
+    private class RemoveReplicationClusters extends CliCommand {
+        @Parameter(description = "tenant/namespace", required = true)
+        private java.util.List<String> params;
```

Comment:
I cannot understand this parameter

Reply:
The `params` represents the `tenant/namespace`, although it is a `List` type but actually only allows `single` value, as below:
https://github.com/apache/pulsar/blob/10818e8e39b9f039793b8534da0252f2f4c91a61/pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CliCommand.java#L47-L48
https://github.com/apache/pulsar/blob/10818e8e39b9f039793b8534da0252f2f4c91a61/pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CliCommand.java#L122-L128

Reply:
@yuruguo I suggest you provide some command-line examples here to describe where is this param place :)